barista-processor/src/test/java/com/markelliot/barista/processor/TestFooServer.java

```diff
@@ -51,7 +51,7 @@ static void afterAll() {
     @Test
     void smokeTest() throws IOException, InterruptedException {
         assertResponse("http://localhost:8282/foo/open/get", 200, "\"Hello, World!\"");
-        assertResponse("http://localhost:8282/foo/open/echo/hellother", 200, "\"hellother\"");
+        assertResponse("http://localhost:8282/foo/open/echo/onetwothreefour", 200, "\"onetwothreefour\"");
```

Comment:
have a suspicion there might be a flush issue for very short responses somewhere